src/main/resources/db/migration/V145__IdentifierSequencesPerPrefix.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE identifier_sequences;
```

Comment:
should be good since we haven't hit prod with https://github.com/terraware/terraware-server/pull/704 yet 👍